cf_xarray/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .accessor import CFAccessor  # noqa
 from .helpers import bounds_to_vertices, vertices_to_bounds  # noqa
+from .options import set_options
```

Comment:
```suggestion
from .options import set_options  # noqa
```

this will silence the flake8 warning